---
 fs/ext4/ext4.h |    6 ++++++
 1 file changed, 6 insertions(+)

--- a/fs/ext4/ext4.h
+++ b/fs/ext4/ext4.h
@@ -2915,6 +2915,8 @@ extern void ext4_mark_bitmap_end(int sta
 extern int ext4_init_inode_table(struct super_block *sb,
 				 ext4_group_t group, int barrier);
 extern void ext4_end_bitmap_read(struct buffer_head *bh, int uptodate);
+extern struct buffer_head *
+ext4_read_inode_bitmap(struct super_block *sb, ext4_group_t block_group);
 
 /* fast_commit.c */
 int ext4_fc_info_show(struct seq_file *seq, void *v);
@@ -3071,6 +3073,10 @@ extern int ext4_ind_migrate(struct inode
 extern int ext4_init_new_dir(handle_t *handle, struct inode *dir,
 			     struct inode *inode,
 			     const void *data1, const void *data2);
+extern struct buffer_head *ext4_append(handle_t *handle,
+				       struct inode *inode,
+				       ext4_lblk_t *block);
+
 extern int ext4_dirblock_csum_verify(struct inode *inode,
 				     struct buffer_head *bh);
 extern int ext4_orphan_add(handle_t *, struct inode *);
